bitcoinspv/bootstrap.go

```diff
@@ -160,19 +160,19 @@ func (r *Relayer) initializeBTCCache() error {
 // waitForBTCToSyncWithNative ensures BTC node is synchronized by checking
 // that its chain height is at least equal to the Native light client height.
```

Comment:
```suggestion
// that its chain height is at least equal to the Bitcoin Light client on Sui

Reply:
I renamed everything to just `light client`. I think its better this way since we may support other networks in the future